src/transient.rs

```diff
@@ -691,3 +691,20 @@ mod either_impls {
         type Transience = (Covariant<L>, Covariant<R>);
     }
 }
+
+#[cfg(feature = "serde_json")]
+mod serde_json_impls {
+    use crate::*;
+    use serde_json::Error;
+
+    impl Static for Error {}
+}
+
+#[cfg(feature = "rmp-serde")]
+mod rmp_serde_impls {
+    use crate::*;
```

Comment:
Same as above, replacing the wildcard import with `use crate::Static` would be preferred.